R/dt_heading.R

```diff
@@ -49,5 +49,9 @@ dt_heading_has_subtitle <- function(data) {
 
   heading <- dt_heading_get(data = data)
 
-  length(heading) > 0 && length(heading$subtitle) > 0 && !is.null(heading$subtitle)
+  length(heading) > 0 &&
+    length(heading$subtitle) > 0 &&
```

Comment:
It would be nice to have the logic for "non-zero-length, non-space-only string" as a utility function

Reply:
Now implemented. Thanks for the suggestion.